bigframes/core/compile/scalar_op_compiler.py

```diff
@@ -1346,6 +1365,25 @@ def clip_op(
         )
 
 
+@scalar_op_compiler.register_nary_op(ops.switch_op)
+def switch_op(*cases_and_outputs: ibis_types.Value) -> ibis_types.Value:
+    # ibis can handle most type coercions, but we need to force bool -> int
```

Comment:
I wonder if it's worth leaving them an issue to allow this implicit cast?

Reply:
maybe? even pandas isn't really consistent on whether bools are numeric or not.